Tests/Integration/MSFT_xNetBIOS.config.ps1

```diff
@@ -0,0 +1,11 @@
+$adapter = (Get-NetAdapter -Physical)[0]
+
+configuration MSFT_xNetBIOS_Config {
+    Import-DscResource -ModuleName xNetworking
+    node localhost {
+        xNetBIOS Integration_Test {
+            InterfaceAlias   = $adapter.Name
+            Setting = 'Disable'
+        }
+    }
+}
```

Comment:
You'll want to add a new line here. [DSCGuidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#general-rules)